configure.ac

```diff
@@ -6,20 +6,20 @@ dnl # Run ./autogen.sh to build configure script
 dnl #
 dnl ##########################################################################
 
-AC_PREREQ(2.59)
+AC_PREREQ(2.63)
```

Comment:
> At least 2.63 is tested, we don't know about the older ones.

This is the argument that persuades me. :-) Better know for sure that what we require works. Commit +1 from me, but please mention it in the commit message.